schemas/user_schema.py

```diff
@@ -25,9 +25,16 @@ class UserCreatedResponse(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
 
+class UserLogin(BaseModel):
+    email: EmailStr
+    password: str
+    role: str = 'user'
```

Comment:
The role is for later implementation of role-based authorization.

Reply:
Can we remove the role and put that in a following PR?

Reply:
why, please?

Reply:
It is for separation of concern, we want to do login not assign roles for now.